native/cocos/renderer/gfx-base/GFXDevice.h

```diff
@@ -48,6 +48,13 @@
 namespace cc {
 namespace gfx {
 
+enum class DeviceState : char {
+    STATE_UNKNOWN = 0,
+    STATE_INITIALIZED,
```

Comment:
It seems STATE_INITIALIZED is not needed. Just need a bool value to check if device is ready for rendering or not?